cmd/livepeer/starter/starter.go

```diff
@@ -712,6 +717,13 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 		go serviceRegistryWatcher.Watch()
 		defer serviceRegistryWatcher.Stop()
 
+		core.PriceFeedWatcher, err = watchers.NewPriceFeedWatcher(backend, *cfg.PriceFeedAddr)
```

Comment:
Won't this fail if someone does not set `priceFeedAddr` flag (or sets the addr for the wrong chain)? I think we should not require setting (correct) `priceFeedAddr` if someone does not want to use price conversion.

Reply:
It won't! I removed the preemptive calls from the `NewPriceFeedWatcher` constructor (`Description()` and `updatePrice()`) to make sure that we wouldn't fail here. Just checked the implementation of the bindings and they also don't make any checks on the provided address before actually using it on a call. I'll double check running these with some invalid address configs, but I believe it should be safe already. Let me know if you think it deserves some explicit logic (or comment?) here though.